docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx

```diff
@@ -97,7 +97,7 @@ let data = await fetch('https://api.vercel.app/blog', { cache: 'no-store' })
 
 ### Fetching data on the server with an ORM or database
 
-This component will fetch and display a list of blog posts. The response from the database will be cached.
+This component will fetch and display a list of blog posts. The response from the database will not be cached.
```

Comment:
Next.js cannot cache database responses automatically. Database calls must be wrapped with the `unstable_cache` for persistent caching or React `cache` for memoization as mentioned in the 
[Caching data with an ORM or Database](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching#caching-data-with-an-orm-or-database) and [Caching data with an ORM or Database](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching#caching-data-with-an-orm-or-database) sections. Hence, correcting this sentence to "The response from the database will not be cached".

Reply:
This will be cached at the page level though so might cause confusion with the comment below saying to avoid caching.

Reply:
How about we change the other sentence to: `"To prevent the page from prerendering, you can include the following in your file:"`?

Reply:
That sounds good although I'd also change this one to something like, "This database call is not included in the data cache by default and will be called whenever the page is rendered."

Reply:
To maintain consistency of wording with the previous section, we could say:  "This component will fetch and display a list of blog posts. The response from the database is not cached by default but could be with [additional configuration](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching#caching-data-with-an-orm-or-database).

Reply:
Sounds good

Reply:
done